### ISSUE TITLE
All Wishlist: broken design

### ISSUE THREAD

**Issue by Fetati** [Goodler/code#502]
Сместились картинки вишлистов при сужении экрана до меньшего размера:
![gooddler gui bug 2](https://cloud.githubusercontent.com/assets/4335901/11989568/df98ba32-a9b4-11e5-94d5-812b7cb1b494.png)

**Comment by DmitrysIF** [comment]
Да, имеется такая проблема. Нужно бы переработать дизайн этой страницы.
